upf/interop/tarski.py

```diff
@@ -133,7 +158,19 @@ def convert_tarski_problem(env: Environment, tarski_problem: tarski.fstrips.Prob
         signature = []
         for t in p.domain:
             signature.append(types[str(t.name)])
-        fluent = upf.model.Fluent(p.name, tm.RealType(), signature)
+        func_sort = p.sort[-1]
+        fluent = None # type: ignore
+        if isinstance(func_sort, Interval):
```

Comment:
As discussed, here we can treat `func_sort == 'Integer'` and `func_sort == 'Real'` as special cases